cli/src/etos_client/downloader/downloader.py

```diff
@@ -142,6 +107,49 @@ def __save_file(self, item: Downloadable, response: requests.Response) -> None:
         with open(download_path, "wb+") as report:
             for chunk in response:
                 report.write(chunk)
+        if not self.__verify(item, download_path):
+            self.logger.error("Checksum verification failed on %r", item)
+
+    def __verify(self, item: Downloadable, path: Path) -> bool:
+        """Verify the checksum of the downloaded item.
+
+        Path is the real path to the saved file.
+        """
+        translation_table = {
```

Comment:
sha256 alone would probably be enough

Reply:
It's enough for ETOS, but with the ETR plugins being able to produce other files for download I think it would be sensible to support all algorithms that are supported by Eiffel, which is the reason I added all of them.